File: Matlab/bench_1_1.m
function bench_1_1() 

trials = 100;
rows = 1000;
cols = 1000;

times = zeros(1,trials);

for i = 1:trials
   M = randi(10000, rows, cols);
   
   tic();
   inv(M);
   t = toc();
   
   times(i) = t;
end

fprintf('Matlab - Benchmark 1.1: Average time = %f milliseconds\n', mean(times)*1000);

File: Python/bench_1_1.py
import numpy as np
from time import time

def bench_1_1():

    trials = 100
    rows = 1000
    cols = 1000

    times = []

    for i in range(trials):
    
        M = np.random.randint(0,10000, size=(rows,cols))

        start = time()
        Minv = np.linalg.inv(M)
        t = time()-start

        times.append(t)

    print 'Python - Benchmark 1.1: Average time = {} milliseconds'.format(np.mean(times)*1000)